typescript/infra/src/roles.ts

```diff
@@ -6,6 +6,8 @@ export enum Role {
   Kathy = 'kathy',
 }
 
+export type FundableRole = Role.Relayer | Role.Kathy;
```

Comment:
ditto, enum?

Reply:
same as above

Reply:
what's your rationale behind it?

Reply:
https://www.becomebetterprogrammer.com/typescript-union-types-vs-enums/
enums provide a better compile time devx but its not a big deal